Doc: Improve formatting and readability for pipeline ordering (#13084)
elastic/logstash: docs/static/settings-file.asciidoc

| `pipeline.ordered`
a|
Set the pipeline event ordering. Valid options are:

> * `auto`. Automatically enables ordering if the `pipeline.workers` setting is also set to `1`.

Reviewer: Should we say what happens if `pipeline.workers` is set to > 1? 

Something like 

```suggestion
* `auto`. Automatically enables ordering if the `pipeline.workers` setting is set to `1`, and disables otherwise
```

> * `auto`. Automatically enables ordering if the `pipeline.workers` setting is set to `1`, and disables otherwise

Author: ```suggestion
* `auto`. Automatically enables ordering if the `pipeline.workers` setting is `1`, and disables otherwise.
```

> * `true`. Enforces ordering on the pipeline and prevent logstash from starting

Author: ```suggestion
* `true`. Enforces ordering on the pipeline and prevents Logstash from starting
```

a|

> Set the pipeline event ordering.Valid options are:

Author: ```suggestion
Set the pipeline event ordering. Valid options are:
```

Author: Fixed pre-existing typo.
